src/engine/MaxConstraint.h

```diff
@@ -160,10 +175,9 @@ class MaxConstraint : public PiecewiseLinearConstraint
     void getEntailedTightenings( List<Tightening> &tightenings ) const override;
 
     /*
-      For preprocessing: get any auxiliary equations that this constraint would
-      like to add to the equation pool.
+      Represent the Max constraint using the aux variables.
     */
-    void addAuxiliaryEquations( InputQuery &inputQuery ) override;
+    void transformToUseAuxVariablesIfNeeded( InputQuery &inputQuery ) override;
```

Comment:
Update function name per the offline discussion.

Reply:
Done